platform.md

```diff
@@ -212,6 +224,19 @@ A platform MAY support any number of mixins for a given stack in order to suppor
 Changes introduced by mixins SHOULD be restricted to the addition of operating system software packages that are regularly patched with strictly backwards-compatible security fixes.
 However, mixins MAY consist of any changes that follow the [Compatibility Guarantees](#compatibility-guarantees).
 
+#### Mixin Resolution
+
+During the prepare phase, a list of provided mixins MUST be retrived from the provided run-image. The analyzer should either fail or ignore the previous image if the run-image is missing any mixins that were present on the previous image.
```

Comment:
> The analyzer 
Again, I'm not sure if this is left over or a word we are using to describe the process happening in the prepare phase.

> provided run-image
Is this coming from the `stack-toml` provided to `preparer`?

Reply:
>> The analyzer

> Again, I'm not sure if this is left over or a word we are using to describe the process happening in the prepare phase.

Fixed

>>provided run-image

>Is this coming from the stack-toml provided to preparer?

I'm not entirely sure. I imagined the `preparer` would need to output a `stack.toml`, but it also needs a stack.toml as input (to get the "provided run-image")